src/services/user.js

```diff
@@ -505,6 +504,141 @@ const updateUser = async (req, res, next) =>
     }
 }
 
+const updateUserToAdmin = async (req, res, next) => {
+    try{
+
+        let userEmail = req.path.split('/');
+        userEmail = userEmail[userEmail.length - 1];
+
+        Logger.write('user', `Update UserToAdmin with email  ${userEmail}`);
+
+        let user = await User.findOne({'email': userEmail});
+        const userId = user._id;
+
+        let isUserExists = await User.findById(userId);
+
+        if (!isUserExists) {
+            return res.status(404).json({
+                'code': 'BAD_REQUEST_ERROR',
+                'description': 'No user found in the system'
+            });
+        }
+
+        const updateAdminStatus = {
+            isAdmin : true
+        }
+
+        let updateUser = await User.findByIdAndUpdate(userId, updateAdminStatus, {
+            new: true // TODO REMOVE THIS
```

Comment:
This TODO seems to  be removed...

Reply:
@janakavishwajith the email sent is already from a list of non admin users that were queried from the database .

Reply:
@janakavishwajith oh okay i get what you mean . will make the changes